manifests/web-cluster/prod/helium/migration-service.yaml

```diff
@@ -26,7 +26,10 @@ spec:
               protocol: TCP
           env:
             - name: NAME
-              value: devnethelium5
+              valueFrom:
+                secretKeyRef:
+                  name: migration-secret
```

Comment:
Why is this a secret?

Reply:
Because someone could frontrun us and take that ID. Worse, issuance and other contracts would listen to their version.